zstacklib/zstacklib/utils/gpu.py

```
@@ -1,6 +1,7 @@
+import os
 import threading
 
-from zstacklib.utils import thread
+from zstacklib.utils import thread, lock
 from zstacklib.utils.bash import *
```

Ruff (0.14.8)

`from zstacklib.utils.bash import *` used; unable to detect undefined names

(F403)